style/pi-hole.css

```diff
@@ -158,27 +158,72 @@
     user-select: none
 }
 
-#chartjs-tooltip {
-    opacity: 1;
+.chartjs-tooltip {
+    opacity: 0;
     position: absolute;
-    background: rgba(0, 0, 0, .7);
-    color: #fff;
-    border-radius: 3px;
-    -webkit-transition: all .1s ease;
-    -o-transition: all .1s ease;
-    transition: all .1s ease;
     pointer-events: none;
-    -webkit-transform: translate(-50%, 0);
-    -ms-transform: translate(-50%, 0);
-    -o-transform: translate(-50%, 0);
-    transform: translate(-50%, 0);
+    color: #fff;
+    background-color: rgba(0,0,0,0.8);
+    max-width: 95%;
+    z-index: 900;
+}
+
+.chartjs-tooltip th {
+    padding-bottom: 3px;
+}
+
+.chartjs-tooltip,
+.chartjs-tooltip .arrow {
+    -webkit-transition: all .2s cubic-bezier(0.165, 0.84, 0.44, 1);
+    transition: all .2s cubic-bezier(0.165, 0.84, 0.44, 1);
```

Comment:
Which properties did you mean to transition? It's better to be explicit, because we might be transitioning properties that are not performant and this makes it harder to notice :)